.bumpversion.cfg

```diff
@@ -3,7 +3,7 @@ commit = True
 tag = True
 sign_tags = True
 message = build: Version {new_version}
-current_version = 1.3.1
+current_version = 1.4.0
```

Comment:
Please update the version compatibility matrix in the README (currently it says Quince should use 1.3).

Reply:
Ack, done.